backend/fms_core/template_importer/row_handlers/sample_qc/sample.py

```diff
@@ -0,0 +1,36 @@
+from datetime import datetime
+
+from fms_core.template_importer.row_handlers._generic import GenericRowHandler
+
+from fms_core.services.sample import get_sample_from_container, update_sample
+from fms_core.services.process_measurement import create_process_measurement
+from fms_core.services.property_value import create_process_properties
+
+class SampleQCRowHandler(GenericRowHandler):
+    def __init__(self):
+        super().__init__()
+
+
+    def process_row_inner(self, sample, sample_information, process_measurement, process_properties):
+        sample, self.errors['sample'], self.warnings['sample'] = get_sample_from_container(
+            barcode=sample['container']['barcode'],
+            coordinates=sample['coordinates'],
+        )
+
+        if sample:
+            #TODO: Update sample with sample_information
+            #TODO: Update derived_sample's flags with sample information
+
+            _, self.errors['process_measurement'], self.warnings['process_measurement'] = \
+                create_process_measurement(
+                    process=process_measurement['process'],
+                    source_sample=sample,
+                    execution_date=process_measurement['execution_date'],
+                    volume_used=process_measurement['volume_used'],
+                    comment=process_measurement['comment'],
+                )
+
+            #TODO: Validate instruments according to platform
+
+            _, self.errors['properties'], properties_warnings = create_process_properties(process_properties,
```

Comment:
Do not forget that properties will be tied to the process_measurement instead of the process as in the experiment_run importer...